src/Tasks/Microsoft.NET.Build.Tasks/SingleProjectInfo.cs

```diff
@@ -92,9 +118,21 @@ public static SingleProjectInfo Create(string projectPath, string name, string f
                 SingleProjectInfo referenceProjectInfo;
                 if (projectReferences.TryGetValue(sourceProjectFile, out referenceProjectInfo))
                 {
-                    ResourceAssemblyInfo resourceAssemblyInfo =
-                        ResourceAssemblyInfo.CreateFromReferenceSatellitePath(projectReferenceSatellitePath);
-                    referenceProjectInfo._resourceAssemblies.Add(resourceAssemblyInfo);
+                    string originalItemSpec = projectReferenceSatellitePath.GetMetadata("OriginalItemSpec");
+
+                    if (!string.IsNullOrEmpty(originalItemSpec))
+                    {
+                        ReferenceInfo referenceInfo = referenceProjectInfo._dependencyReferences.SingleOrDefault(r => r.FullPath.Equals(originalItemSpec));
+
+                        if (referenceInfo is null)
+                        {
+                            // We only want to add the reference satellite path if it isn't already covered by a dependency
```

Comment:
Do we have test coverage for this satellite path change?

Reply:
No, looks like I forgot to push those, let me see if I still have them locally.

Reply:
Fixed.